features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -817,6 +817,14 @@
 				<AttributeID>failedBackupCodeAttempts</AttributeID>
 				<Description>Claim to store failed Backup Code Attempts</Description>
 			</Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/identity/otpbackupcodes</ClaimURI>
+				<DisplayName>OTP Backup Codes</DisplayName>
+				<!-- If user store based identity store is used, a proper mapped attribute
+                in your user store must be configured for this. -->
+				<AttributeID> OTP backupCodes</AttributeID>
+				<Description>Claim to store OTP backup codes of the user</Description>
+			</Claim>
```

Comment:
Shall we avoid adding this claim by default? Because there is a new identity claim as well for backup codes in the new implementation and this could get confused with it.

(the new claim can not be used with older implementation since new one stores the backup codes in encrypted form)

Reply:
Addressed in [90c019e](https://github.com/wso2/carbon-identity-framework/pull/5155/commits/90c019efc7acd8f62b533f94efdfe6eed4812701)